ext/opentelemetry-ext-opentracing-shim/src/opentelemetry/ext/opentracingshim/__init__.py

```diff
@@ -0,0 +1,257 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from contextlib import contextmanager
+
+import opentracing
+
+from opentelemetry.ext.opentracingshim import util
+
+logger = logging.getLogger(__name__)
+
+
+def create_tracer(tracer, scope_manager=None):
```

Comment:
Actually we can't allow users to provide a `ScopeManager` here, as it will mess up with the context propagation done at the OTel level. So I'd remove it from now while we figure out what to do when specifically working with non-supported frameworks (such as `gevent`).

Reply:
Done. @carlosalberto could you please open a follow up issue for this? I would have done it myself but I'm not sure I can phrase the problem correctly.